client/lib/credit-card-details/test/index.js

```diff
@@ -1,15 +1,13 @@
 /** @format */
-
 /**
  * External dependencies
  */
-
 import assert from 'assert';
 
 /**
  * Internal dependencies
  */
-import creditCardDetails from '../';
+import { getCreditCardType } from '../index';
```

Comment:
Not important, but I thought we preferred `'../'` over `'../index'`?

Reply:
yeah probably. I didn't know if we did or not, but this worked better in my IDE… I was lazy